feat(helm-chart): make charts Openshift compliant (#3415)
keptn/lifecycle-toolkit: docs/docs/installation/configuration/Openshift.md

---
comments: true
---

> # Installing on Openshift

Reviewer: ```suggestion
# Installing on OpenShift
```

Author: I don't think [OpenShift is an approved word, only Openshift.](https://github.com/keptn/lifecycle-toolkit/blob/c5d2b99d4501de5c59e49486a8f636566d88f1b7/.github/actions/spelling/expect.txt#L467)

Do you want me to change that?

Reviewer: you are right! please ignore the suggestions

> To install on Openshift, set the value `global.isOpenshift` in the `values.yaml` file to true.

Reviewer: ```suggestion
To install on Openshift, set the value `global.openShift.enabled` in the `values.yaml` file to true.
```

In practice this means that `runAsUser` and `runAsGroup` are removed, since
Openshift sets those automatically.

> You can set the `global.isOpenshift` parameter when running the `helm install` command:

Reviewer: ```suggestion
You can set the `global.openShift.enabled` parameter when running the `helm install` command:
```

```shell

> helm install keptn keptn --set global.openShift.enabled=true

Reviewer: ```suggestion
helm install keptn keptn/keptn -n keptn-system --create-namespace --set global.openShift.enabled=true
```
